<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0">
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="m_screenshotImage">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>160</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"> QLabel {
     background-image: url(:/DesktopCapture/striped-background.png);
 }</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="m_controls" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
       <item>
        <widget class="QGroupBox" name="m_captureGroupBox">
         <property name="styleSheet">
          <string notr="true"> QGroupBox {
     border: 1px solid gray;
     border-radius: 5px;
     margin-top: 1ex; /* leave space at the top for the title */
 }

 QGroupBox::title {
     subcontrol-origin: margin;
     subcontrol-position: top center; /* position at the top center */
     padding: 0 3px;
 }</string>
         </property>
         <property name="title">
          <string>Desktop Capture</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="m_captureAllMonitors">
            <property name="text">
             <string>Capture all monitors</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="m_captureMonitor">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Capture monitor</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="m_captureMonitorComboBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="m_cameraEnabled">
            <property name="text">
             <string>Enable Camera</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Camera Resolution</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="m_cameraResolutionComboBox"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Composition Mode</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="m_compositionComboBox"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Overlay Position</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="m_cameraPositionComboBox"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="m_screenshotAnimateTray">
            <property name="text">
             <string>Animated tray icon</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_captureVideo">
            <property name="text">
             <string>Capture Video</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Video Quality</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="m_captureVideoQuality"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="1,0">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Time between captures</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="m_screeshotTime">
              <property name="time">
               <time>
                <hour>0</hour>
                <minute>0</minute>
                <second>30</second>
               </time>
              </property>
              <property name="maximumTime">
               <time>
                <hour>0</hour>
                <minute>59</minute>
                <second>59</second>
               </time>
              </property>
              <property name="minimumTime">
               <time>
                <hour>0</hour>
                <minute>0</minute>
                <second>5</second>
               </time>
              </property>
              <property name="displayFormat">
               <string>mm:ss</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Output Directory</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="m_dirEditLabel">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="m_dirButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="m_pomodoroGroupBox">
         <property name="styleSheet">
          <string notr="true"> QGroupBox {
     border: 1px solid gray;
     border-radius: 5px;
     margin-top: 1ex; /* leave space at the top for the title */
 }

 QGroupBox::title {
     subcontrol-origin: margin;
     subcontrol-position: top center; /* position at the top center */
     padding: 0 3px;
 }</string>
         </property>
         <property name="title">
          <string>Pomodoro Configuration</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,0">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Pomodoro</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="m_pomodoroTime">
              <property name="time">
               <time>
                <hour>0</hour>
                <minute>25</minute>
                <second>0</second>
               </time>
              </property>
              <property name="maximumTime">
               <time>
                <hour>0</hour>
                <minute>59</minute>
                <second>0</second>
               </time>
              </property>
              <property name="minimumTime">
               <time>
                <hour>0</hour>
                <minute>5</minute>
                <second>0</second>
               </time>
              </property>
              <property name="displayFormat">
               <string>mm:ss</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,0">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Short break</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="m_shortBreakTime">
              <property name="time">
               <time>
                <hour>0</hour>
                <minute>5</minute>
                <second>0</second>
               </time>
              </property>
              <property name="maximumTime">
               <time>
                <hour>0</hour>
                <minute>59</minute>
                <second>0</second>
               </time>
              </property>
              <property name="minimumTime">
               <time>
                <hour>0</hour>
                <minute>1</minute>
                <second>0</second>
               </time>
              </property>
              <property name="displayFormat">
               <string>mm:ss</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,0">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Long break</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="m_longBreakTime">
              <property name="time">
               <time>
                <hour>0</hour>
                <minute>15</minute>
                <second>0</second>
               </time>
              </property>
              <property name="maximumTime">
               <time>
                <hour>0</hour>
                <minute>59</minute>
                <second>0</second>
               </time>
              </property>
              <property name="minimumTime">
               <time>
                <hour>0</hour>
                <minute>1</minute>
                <second>0</second>
               </time>
              </property>
              <property name="currentSection">
               <enum>QDateTimeEdit::MinuteSection</enum>
              </property>
              <property name="displayFormat">
               <string>mm:ss</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,0">
            <item>
             <widget class="QLabel" name="m_pomodoroBreak">
              <property name="text">
               <string>Pomodoros before a long break</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="m_pomodorosBreakNumber">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>10</number>
              </property>
              <property name="value">
               <number>4</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="m_pomodoroAnimateTray">
            <property name="text">
             <string>Animated tray icon</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_pomodoroUseSounds">
            <property name="text">
             <string>Use sounds</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_continuousTicTac">
            <property name="text">
             <string>Continuous Tic-Tac</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,0">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Number Of Pomodoros Per Session</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="m_pomodorosNumber">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>99</number>
              </property>
              <property name="value">
               <number>12</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_15">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Initial task</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="m_pomodoroTask">
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="m_taskEditButton">
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="statusTip">
               <string>Modify initial task name</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="rsc/resources.qrc">
                <normaloff>:/DesktopCapture/pen.png</normaloff>:/DesktopCapture/pen.png</iconset>
              </property>
              <property name="popupMode">
               <enum>QToolButton::InstantPopup</enum>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="m_startButton">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="text">
         <string>Start!</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>793</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="rsc/resources.qrc"/>
 </resources>
 <connections/>
</ui>
